browser.js

```diff
@@ -79,6 +88,31 @@ function registerShortcuts(username) {
 		return false;
 	});
 
+	// vim bindings
+	Mousetrap.bind('j', () => {
+		scrollToY(fromScrollTop(scrollStep));
+	});
+
+	Mousetrap.bind('k', () => {
+		scrollToY(fromScrollTop(scrollStep * -1));
+	});
+
+	Mousetrap.bind('g g', () => {
+		scrollToY(0);
+	});
+
+	Mousetrap.bind('ctrl+d', () => {
+		scrollToY(fromScrollTop(window.innerHeight * 0.9));
```

Comment:
0.9 is just more in line with how page up/down normally work, ie not doing 100% pagination. But yes it is unclear, I have added a named variable for this.